src/Components/MobileNavbar/MobileNavbar.tsx

```diff
@@ -0,0 +1,33 @@
+import classes from './MobileNavbar.module.css'
+import {faHome,faHistory,faBookOpen,faBook} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome'
```

Comment:
One question why are you using Fontawesome when you are using material UI?

Reply:
I couldn't find the icons I was looking for in material icon so i went with fontawesome here